openedx/core/djangoapps/models/course_details.py

```diff
@@ -103,36 +103,36 @@ def fetch(cls, course_key):
         return cls.populate(modulestore().get_course(course_key))
 
     @classmethod
-    def populate(cls, course_descriptor):
+    def populate(cls, course_block):
```

Comment:
Nit: maybe let's just call it a `block`, as this method is full of long and barely readable lines.

Reply:
Done.